### DISCUSSION TITLE
TagAnnotation @Tag use

### DISCUSSION THREAD

**Discussion by BritneyStars** [apache/skywalking#7584] [Q&A] [answered]
I found a question about @Tag. When use @Tag alone, the custom tag value will not be recorded to the segment. For example, 
in springboot project, 
@Scheduled(cron="0 * * * * ?)
@Tag(key="code",value="returndObj.code")
public ResultJson test(){
   ......
  return new ResultJson("001")
}
In segment indice, the operationname records do not contain the code field.  I want to why this? I debug agent local , find when trigger test() method , it will not enter the beforeMethod() and afterMethod() in TagAnnotationMetheodInterceptor class

**Comment by wu-sheng** [comment, marked as answer]
I think this is caused by plugins confliction. 2 plugins(`scheduled` and `tag`) are both required to work. But right now, there is only one could work. 

Read this discussion, https://github.com/apache/skywalking/discussions/7491. There was a long discussion about this.

**Reply by BritneyStars** [reply]
Will the conflict be resolved later, so will both work?

**Reply by wu-sheng** [reply]
I have given you the context and latest status.
